cadence/contracts/bridge/FlowEVMBridgeOptOut.cdc

```diff
@@ -0,0 +1,10 @@
+/// This contract defines a simple interface which can be implemented by any resource to prevent it from being
+/// onboarded to the Flow-EVM bridge
+///
+access(all) contract FlowEVMBridgeOptOut {
+    /// Implementing this interface in your resource will prevent it from being onboarded to the Flow-EVM bridge
+    /// NOTE: This is suggested only for cases where your asset (NFT/FT) incorporates non-standard logic that would
+    ///      break your project if not handles properly
+    ///      e.g. assets are reclaimed after a certain period of time, NFTs share IDs, etc.
+    access(all) resource interface Asset {}
```

Comment:
Should this be renamed to be a bit more specific? I.e. NonEVMBridgeableAsset or something that sounds nicer than my suggestion?

Reply:
In retrospect yeah, `Asset` is about as vague as it gets. I was thinking of it in the context of the fully referenced name outside of this contract `FlowEVMBridgeOptOut.Asset`. I'll revisit this one